brotab/api.py

```diff
@@ -97,7 +97,7 @@ def activate_tab(self, args):
         #self._get('/activate_tab/%s' % strWindowTab)
 
     def get_active_tab(self, args) -> str:
```

Comment:
```suggestion
    def get_active_tabs(self, args) -> [str]:
```

Reply:
maybe there are other users for this method, it needs to be checked

Reply:
```
Searching 51 files for "get_active_tab" (regex)

/brotab/brotab/api.py:
   97          #self._get('/activate_tab/%s' % strWindowTab)
   98  
   99:     def get_active_tab(self, args) -> str:
  100:         return [self.prefix_tab(tab) for tab in self._get('/get_active_tab').split(',')]
  101  
  102      # def new_tab(self, prefix, search_query):
  ...
  233              api.activate_tab(args)
  234  
  235:     def get_active_tabs(self, args):
  236:         return [api.get_active_tab(args) for api in self._apis]
  237:         #return ['%s\t%s' % (api.get_active_tab(args), api) for api in self._apis]
  238  
  239      # def new_tab(self, args):

/brotab/brotab/main.py:
  151      apis = create_clients()
  152      # api = MultipleMediatorsAPI(create_clients())
  153:     # tabs = api.get_active_tabs(args)
  154      for api in apis:
  155:         tabs = api.get_active_tab(args)
  156          for tab in tabs:
  157              print('%s\t%s' % (tab, api))

/brotab/brotab/extension/background.js:
  399    }
  400  
  401:   else if (command['name'] == 'get_active_tab') {
  402      console.log('Getting active tab');
  403      getActiveTab();

/brotab/brotab/extension/chrome/background.js:
  399    }
  400  
  401:   else if (command['name'] == 'get_active_tab') {
  402      console.log('Getting active tab');
  403      getActiveTab();

/brotab/brotab/extension/firefox/background.js:
  399    }
  400  
  401:   else if (command['name'] == 'get_active_tab') {
  402      console.log('Getting active tab');
  403      getActiveTab();

/brotab/brotab/mediator/brotab_mediator.py:
  122          self._transport.send(command)
  123  
  124:     def get_active_tab(self) -> str:
  125          logger.info('getting active tab')
  126:         command = {'name': 'get_active_tab'}
  127          self._transport.send(command)
  128          return self._transport.recv()
  ...
  200  
  201  
  202: @app.route('/get_active_tab')
  203: def get_active_tab():
  204:     return browser.get_active_tab()
  205  
  206  
  ...
  248  #    - /new_tab (google search)
  249  #    - /get_tab_text
  250: #    - /get_active_tab_text
  251  #
  252  # TODO: fix bug when the number of tabs > 1100

16 matches across 6 files
```

It seems there are no other invocations of the method, but it might be beneficial to pluralize related names in `brotab_mediator.py` and `*/background.js` files so the convention is uniform.

Reply:
> beneficial to pluralize related names

yep, agree on this